rllib/policy/torch_policy.py

```diff
@@ -151,7 +151,26 @@ def __init__(
         else:
             logger.info("TorchPolicy (worker={}) running on {} GPU(s).".format(
                 worker_idx if worker_idx > 0 else "local", config["num_gpus"]))
-            gpu_ids = ray.get_gpu_ids()
+            # We are a remote worker (WORKER_MODE=1):
+            # GPUs should be assigned to us by ray.
+            from ray.worker import global_worker
+            if global_worker.mode == 1:
```

Comment:
global_worker can be None sometimes right? 

also, can we use the "WORKER_MODE" enum?

Reply:
done.